README.md

```diff
@@ -3,30 +3,85 @@
 ![Tier1 Website Template](./assets/Tier1_Website_Template.gif)
 
 ## Overview
-This project provides you with Great Practice Making MODERN LOOKING web apps by cloning the "Conquer" Website Template
-in HTML/CSS/JS WITHOUT the Bootstrap and Jquery or other libraries (like Materialize)!
+This project provides you with Great Practice Making MODERN LOOKING web apps 
+by cloning the "Conquer" Website Template in HTML/CSS/JS WITHOUT Bootstrap, 
+jQuery, or other libraries (like Materialize)!
 
-When you download it to check out, the design is made using Bootstrap and Jquery, but we want you to make it without
-those libraries, just using your mad HTML, CSS, and JavaScript skillz!
+When you download it to check out, the design is made using Bootstrap and 
+jQuery, but we want you to make it without those libraries, just using your 
+mad HTML, CSS, and JavaScript skillz!
 
-When you are done, this type of template looks terrific and gracefully responds on all devices. You could make different 
-versions to use in future web apps you develop!
+When you are done, this type of template looks terrific and gracefully 
+responds on all devices. You could make different versions to use in future 
+web apps you develop!
 
 ## Instructions
 
-For this project you must use Native (Vanilla/Plain) HTML5, CSS, and JavaScript for this project. You are encouraged to use
-CSS-Grid, Flexbox, Media Queries, a naming system like BEM, semantic HTML tags, or other modern techniques! You may not use frameworks or libraries such as Bootstrap.
+For this project you must use Native (Vanilla/Plain) HTML5, CSS, and JavaScript
+for this project. You are encouraged to use CSS-Grid, Flexbox, Media Queries, a
```

Comment:
omit the 'for this project' at the end of the sentence as the sentence begins with the same phrase.

Reply:
Resolved